tests/test_unix_socket.py

```diff
@@ -179,6 +179,7 @@ async def client(app: Sanic, loop: AbstractEventLoop):
             assert r.status_code == 200
             assert r.text == os.path.abspath(SOCKPATH)
     finally:
+        await sleep(0.2)
```

Comment:
Yeah, since this test is using the async client (so that it does both sides on the same loop), if there is not a delay it proceeds to `stop` before the request handler ever executes. `0.1` probably is fine too, but I figure there is little to no difference for us humans, and that `0.1` might solve some headaches in CI.